editor/scene_tree_dock.cpp

```diff
@@ -377,16 +376,16 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 		} break;
 		case TOOL_CLEAR_SCRIPT: {
 
-			Array selection = editor_selection->get_selected_nodes();
+ 			Array selection = editor_selection->get_selected_nodes();
 
 			if (selection.empty())
 				return;
 
 			editor_data->get_undo_redo().create_action(TTR("Clear Script"));
 			editor_data->get_undo_redo().add_do_method(editor, "push_item", (Script *)NULL);
 
-			for (int i = 0; i < selection.size(); i++) {
-
+ 			for (int i = 0; i < selection.size(); i++) {
+				 
```

Comment:
Trailing whitespace on this line as well.

On the two changed lines above, there is an addition of a single space before the tabs used for indentation.